usaspending_api/awards/views.py

```diff
@@ -67,6 +62,9 @@ def get_queryset(self):
         if not (fiscal_year and awarding_agency_id):
             raise InvalidParameterException('Missing one or more required query parameters: fiscal_year, awarding_agency_id')
 
+        if not check_valid_toptier_agency(awarding_agency_id):
```

Comment:
If the agency id does not exist in the db, calling check_valid_toptier_agency will throw a Null pointer error because it calls .first()